quickstarts/microsoft.maps/maps-use-sas/metadata.json

```diff
@@ -5,6 +5,6 @@
   "description": "This template deploys and Azure Maps account and lists a Sas token based on the provided User Assigned identity to be stored in an Azure Key Vault secret.",
   "summary": "This template deploys and Azure Maps account and lists a Sas token and creates an Azure Key Vault secret with the sas token as the value.",
   "githubUsername": "stack111",
-  "docOwner": "philmea",
-  "dateUpdated": "2021-10-24"
+  "docOwner": "eriklindeman",
```

Comment:
Shouldn't the `docOwner` be _stack111_?

Reply:
docOwner can review the PR, so I assign it to Erik. I'm ok to assign to either `stack111` or `eriklindeman`.

Reply:
Updated